lib/gracenote/request/fetch.rb

```diff
@@ -0,0 +1,14 @@
+require "gracenote/request/option"
+
+module Gracenote
+  module Request
+    module Fetch
```

Comment:
__[rubocop]__
* module definition in method body
(Using Ruby 2.0 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)